adafruit_platformdetect/board.py

```diff
@@ -73,6 +73,8 @@ def id(self):
             board_id = self._pynq_id()
         elif chip_id == chips.A64:
             board_id = self._pine64_id()
+        elif chip_id == chips.A33:
+            board_id = boards.CLOCKWORK_PI
```

Comment:
you should attempt a more specific test here

Reply:
I was thinking about that but since it was the only board I went with the simple variant. I'll add a function for it.

Reply:
Fixed and made some other various improvements as well.